addon/webextension/background/startBackground.js

```diff
@@ -60,16 +51,12 @@ this.startBackground = (function() {
     let hasSeenOnboarding = !!result.hasSeenOnboarding;
     if (!hasSeenOnboarding) {
       let path = "icons/icon-starred-32-v2.svg";
-      if (!usePhotonPageAction) {
-        browser.browserAction.setIcon({path});
-      } else {
-        iconPath = path;
-        if (photonPageActionPort) {
-          photonPageActionPort.postMessage({
-            type: "setProperties",
-            iconPath
-          });
-        }
+      iconPath = path;
```

Comment:
Nit: I think you can remove `path` now and just set `iconPath` to the path string directly.